source/percentile.js

```diff
@@ -0,0 +1,56 @@
+import { curry, sort, length, filter} from "ramda"
+
+import isNumeric from "./internal/isNumeric"
+
+/**
+ * Percentile of a value in a series of series using midpoint interpolation. see https://numpy.org/doc/stable/reference/generated/numpy.percentile.html
+ *
+ * @func
+ * @memberOf Z
+ * @category Analysis
+ * @param {Number} percentile to calculate
```

Comment:
Suggest we state in the doc string something like "percentile to calculate in decimal form (eg 0.5 for 50th percentile, 0.75 for 75th percentile)" and update the below accordingly. to handle only one format (either integer or decimal).

Reply:
(see comment below)